src/js/utilities/date/index.js

```diff
@@ -38,46 +43,38 @@ function formatDiff(diff, desc) {
   return `${diff} ${desc}${diff === 1 ? '' : 's'}`;
 }
 
-export function dateToMoment(dateField) {
-  return moment({
-    year: dateField.year.value,
-    month: dateField.month.value ? parseInt(dateField.month.value, 10) - 1 : '',
-    day: dateField.day ? dateField.day.value : null
-  });
-}
-
 /**
  * timeFromNow returns the number of days, hours, or minutes until
  * the provided date occurs. It’s meant to be less fuzzy than moment’s
  * timeFromNow so it can be used for expiration dates
  *
- * @param date {Moment Date} The future date to check against
- * @param userFromDate {Moment Date} The earlier date in the range. Defaults to today.
+ * @param date {Date} The future date to check against
+ * @param userFromDate {Date} The earlier date in the range. Defaults to today.
  * @returns {string} The string description of how long until date occurs
  */
 export function timeFromNow(date, userFromDate = null) {
```

Comment:
Just noting for future optimization that the only place where we use `timeFromNow` (and thus all the `differenceIn*` from `date-fns` is the "Please don't push Submit so often" message. Subtracting the two dates and expressing in terms of integer minutes is probably all we need there. I think we should land this as-is though and work on these details later.